src/exec/processor.rs

```diff
@@ -406,11 +406,11 @@ impl<'a> ExprCompiler<'a> {
     let builder = &self.builder;
 
     self.stack.push(match *op {
-      ArithmOp::Plus => {builder.create_add(&lhs_val, &rhs_val)}
+      ArithmOp::Add => {builder.create_add(&lhs_val, &rhs_val)}
```

Comment:
It need to be a separate issue because it seems to be not related to this issue.